catt/controllers.py

```diff
@@ -26,6 +26,20 @@ def get_stream_info(video_url):
     return stream_info
 
 
+def get_chromecast(device_name):
+    devices = pychromecast.get_chromecasts()
+    if device_name:
```

Comment:
Why not something like this?:

~~~.python
# Sort consistently.
devices = sorted(pychromecast.get_chromecasts(), key=lambda cc: cc.name)
if device_name:
    devices = [device for device in devices if device.name == device_name]
if not devices:
    raise CattCastError("No suitable device found.")
return devices[0]
~~~